packages/next-templates/src/app/meldingen/verhuizing-doorgeven/page.tsx

```diff
@@ -0,0 +1,398 @@
+                },
+              ]}
+            />
+
+            <Heading2>Wie kan de verhuizing doorgeven?</Heading2>
+            <UnorderedList>
+              <UnorderedListItem>Iedereen van 16 jaar of ouder.</UnorderedListItem>
+              <UnorderedListItem>Ouders, voogden of verzorgers voor kinderen jonger dan 18 jaar.</UnorderedListItem>
+              <UnorderedListItem>
+                Woont u op hetzelfde adres en verhuist u naar hetzelfde nieuwe adres? Dan kunt u de verhuizing doorgeven
+                voor:
+                <UnorderedList>
+                  <UnorderedListItem>uw ouders en uw meerderjarige kind of kinderen.</UnorderedListItem>
+                  <UnorderedListItem>uw echtgenoot of geregistreerd partner.</UnorderedListItem>
+                </UnorderedList>
+              </UnorderedListItem>
+            </UnorderedList>
+
+            <Heading2>Verhuizing doorgeven</Heading2>
+            <Paragraph>
+              We kunnen uw verhuizing alleen verwerken als uw aanvraag compleet is. Soms mist er nog iets. Dan nemen we
+              contact met u op. Daarom is het handig als u bij uw aanvraag uw telefoonnummer doorgeeft.
+            </Paragraph>
+
+            <Heading3>Online</Heading3>
+            <Paragraph>U hebt nodig:</Paragraph>
+
+            <UnorderedList>
+              <UnorderedListItem>
+                <Link href="https://www.digid.nl/">Uw Digid</Link>
+              </UnorderedListItem>
+              <UnorderedListItem>Als u verhuist naar een huurwoning of koopwoning:</UnorderedListItem>
+              <UnorderedList>
+                <UnorderedListItem>kopie van uw huurcontract of eigendomsbewijs</UnorderedListItem>
+              </UnorderedList>
+              <UnorderedListItem>Als u verhuist naar een instelling:</UnorderedListItem>
+              <UnorderedList>
+                <UnorderedListItem>
+                  uw overeenkomst met de instelling of een verklaring van de instelling
+                </UnorderedListItem>
+              </UnorderedList>
+              <UnorderedListItem>Als u bij iemand anders gaat wonen:</UnorderedListItem>
+              <UnorderedList>
+                <UnorderedListItem>
+                  het ingevulde en met pen ondertekende formulier{' '}
+                  <Link href="https://www.utrecht.nl/fileadmin/uploads/documenten/9.digitaalloket/Burgerzaken/Formulier-Verklaring-bewoning-adres.pdf">
+                    Verklaring van inwoning
+                  </Link>{' '}
+                  (pdf, 557 kB).
+                </UnorderedListItem>
+                <UnorderedListItem>
+                  kopie van een geldig identiteitsbewijs van de persoon bij wie u gaat wonen.
+                </UnorderedListItem>
+              </UnorderedList>
+            </UnorderedList>
+
+            <Heading3>Met DigiD</Heading3>
+            <ButtonGroup className="voorbeeld-button-group">
```

Comment:
Waarom is deze class name nodig? `voorbeeld-button-group`

Reply:
Deze heb ik toegevoegd om een inline gap 0 te zetten op de button en DigiD logo, omdat er een groote gap tussen in zat